README.md

```diff
@@ -4,6 +4,7 @@
 
 | JDK version                | Docker tags                                               | Dockerfile                                                 |
 | -------------------------- | --------------------------------------------------------- | ---------------------------------------------------------- |
+| openjdk-8( 8u222b10 ) + Ubuntu 18.04 | `openjdk-8`, `openjdk-8-ubuntu`, `openjdk-8-ubuntu-18`, `openjdk-8-ubuntu-1804` | *[( openjdk-8/ubuntu/18.04/Dockerfile )](./openjdk-8/ubuntu/18.04/Dockerfile)* |
```

Comment:
Please format the table to have aligned columns. You can use the ``Markdown All in One`` vscode plugin if you want